packages/mobile/src/invite/saga.ts

```diff
@@ -217,7 +220,13 @@ export function* redeemInviteSaga(action: RedeemInviteAction) {
 
     // Check that the balance of the new account is not 0
     const StableToken = yield call(getStableTokenContract, web3)
-    const stableBalance = new BigNumber(yield call(StableToken.methods.balanceOf(tempAccount).call))
+    Logger.debug(TAG + '@redeemInviteCode', 'web3 is: ' + (yield select(web3ReadySelector)))
+
+    // Try to get balance once + two retries before failing
+    const stableBalance = new BigNumber(
+      yield call(reTryAsync, StableToken.methods.balanceOf(tempAccount).call, 2, [])
```

Comment:
I don't think that adding retry logic is the way to go. We'd have to add it everywhere to be consistent and I'm not sure it helps chances of success here.